minio/lifecycleconfig.py

```diff
@@ -246,6 +246,8 @@ def __init__(self, status, abort_incomplete_multipart_upload=None,
                  noncurrent_version_transition=None,
                  transition=None):
         check_status(status)
+        if not rule_filter:
```

Comment:
Maybe we set rule_filter to empty string ?

Reply:
i don't think we need to assume this. It depends on `action`